docs/docs/requirements.md

```diff
@@ -0,0 +1,21 @@
+---
+slug: /hardware-requirements
+sidebar_position: 5
+title: Juno Full Node Hardware Requirements
+---
+
+This outlines the hardware specifications required to run a Juno full node, including both minimal and recommended requirements.
+
+## Minimal Hardware Specifications
+
+- **CPU**: 2+ cores
+- **RAM**: 4GB+
+- **Storage**: 200GiB (SSD recommended; note: storage requirements will increase over time)
```

Comment:
What is difference between 200GiB and 200GB?

Reply:
https://massive.io/file-transfer/gb-vs-gib-whats-the-difference/